tsqc/model.py

```diff
@@ -442,14 +442,12 @@ def nodes_df(self):
         child_left, child_right = self.child_bounds(
             ts.num_nodes, ts.edges_left, ts.edges_right, ts.edges_child
         )
```

Comment:
Changing these data frames is tricky now because of our cache.  Any time we add or delete a column, we will need to increment the version string above. So here, you need to change ``  @disk_cache("v1")`` to   @disk_cache("v2")